src/AbstractZone.php

```diff
@@ -85,4 +85,12 @@ protected function getZonePath(?string $path = null) : string
     {
         return sprintf('zones/%s%s', $this->zone, $path);
     }
+
+    /**
+     * @return string
+     */
+    public function getCanonicalName(): string
+    {
+        return substr($this->zone, 0, strlen($this->zone) - 1);
```

Comment:
A canonical name _must_ include the `.` at the end. Either change this line or rename the method.

```suggestion
        return $this->zone;
```